image-processing/scoreDetectionTest.py

```diff
@@ -0,0 +1,97 @@
+import cv2
```

Comment:
Lowkey ignore this file, its only used for testing image processing on single images.